guides/v2.0/cloud/live/go-live-checklist.md

```diff
@@ -55,13 +55,10 @@ The following is a possible list of DNS providers strictly for your information.
 
 Many other providers also offer workarounds to accomplish this goal. The most common is to add a CNAME record for the `www` host on the domain and then use the DNS provider's redirection service to redirect the apex over to the `www` version of the domain. Consult your DNS provider to see how they support this.
 
-For the format, we support `www.domain.tld CNAME <environment>-<project>.<region>.magentosite.cloud`. We don't support a domain without www: `domain.tld CNAME <environment>-<project>.<region>.magentosite.cloud`.
-
-### Configure DNS for Fastly {#fastly-dns}
-We provide a Domain Validated SSL certificate with Subject Alternative Name enabled, issued by GLobalSign. The domain validation process is executed by Fastly. When you are ready to go, you will need to provide your domain names in a Support ticket to us. We will then provide a DNS TXT record to add to your apex domain to confirm ownership.
+Another option for <em>apex domain</em> is to add an A record which maps a domain name to the Fastly IP address: `150.101.113.124`.
```

Comment:
Is this IP address the same for everyone or is there a more generic way to present this, such as where to find the Fastly IP address?

Reply:
@hguthrie This IP is the same for every Magento Cloud customer, it's not a mistake. Actually, it's the IP address where the DNS name prod.magentocloud.map.fastly.net points to. Despite it looks like a single host and single point of failure, it isn't, because uses an IP multicast technology.